docs/CHANGELOG.asciidoc

```diff
@@ -35,6 +35,7 @@
 * Upgrade PyTorch to version 1.11. (See {ml-pull}2233[#2233], {ml-pull}2235[#2235]
   and {ml-pull}2238[#2238].)
 * Upgrade zlib to version 1.2.12 on Windows. (See {ml-pull}2253[#2253].)
+* Add control message to dynamically change the number of model allocations. (See {ml-pull}2258[#2258].)
```

Comment:
We should think carefully about how we want this to look in the release notes. We don't want both the C++ and Java PRs to appear as separate items. If the Java PR is going to make it clearer to the user how they would use it then maybe just that one should appear and this PR should be `>non-issue`. Or otherwise the Java PR should be a `>non-issue`.

Reply:
I've changed my mind on this because it is an internal change to enable another feature. In fact even the Java side of this might not be release noted as that is also part of the change to improve model allocations